src/backend/main.mo

```diff
@@ -466,4 +504,99 @@ actor {
     return "Hello, " # name # "! " # "Your PrincipalId is: " # Principal.toText(caller);
   };
 
+  // Images
+
+  public shared (msg) func uploadImg(imgId : ImgId, image : Blob) {
+    storeBlobImg(imgId, image);
```

Comment:
nemal by aj tu byť ten if že túto metódu nemôže zavolať hocikto?

Reply:
Good point, tuto veřejnou už ani nepoužívám, volá se interně v createProduct a updateProduct, dám ji pryč.

Pokud zbude čas, tak do těch metod, přidám isAdmin ochranu, chtěl jsem to deploynout tak, aby se každý přihlášený mohl podívat do adminu v rámci dema, ale provádět změny by mohl jen admin (my).

Reply:
Good point, tuto veřejnou už ani nepoužívám, volá se interně v createProduct a updateProduct, dám ji pryč.

Pokud zbude čas, tak do těch metod, přidám isAdmin ochranu, chtěl jsem to deploynout tak, aby se každý přihlášený mohl podívat do adminu v rámci dema, ale provádět změny by mohl jen admin (my).